projects/syntheticBabylon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+authors:
+  - Theodor Hillmann
+authors-url:
+  - "http://theodorhillmann.de"
+layout: project
+title: "Synthetic Babylon: analogue generative architecture"
+type: project
+---
+
+# Synthetic Babylon
+## analogue generative architecture
+
+This work is part of a class at the University of Applied Sciences Potsdam. The project engages in analogue generative design. Initially an analogue algorithm has been written. I tried to generate architectural sketches inspired by Constant's "New Babylon". This initially analog algorithm has been extended and developed over many iterations and finally put in form.
+
+For the algorithms first iteration pencils and sheets of paper were handed to 5 people. They were instructed to consequently follow the rules of the algorithm listed below. The following "code" is kept in written-german to stay closest to the original experience.
+
+### Iteration 1:
+
+Arbeite die folgenden Punkte nacheinander ab.
+Nutze dafür ein blankes Blatt Papier und ein Bleistift (wenn
+möglich).
+
+1. Zeichne zwei Punkte die waagerecht zueinander stehen, und
```

Comment:
Maybe also make them a Markdown quote. All the other students did this as well. Just add to each line at the beginning a 

    >